### PR TITLE
docs($animateCss): add missing doc

### LINKED ISSUE #13049: Add documentation for structural option in animateCss service
The `structural` option for `$animateCss` service is undocumented on the service page (https://docs.angularjs.org/api/ngAnimate/service/$animateCss), but is used in the animation examples (https://docs.angularjs.org/api/ngAnimate).

To find out what this option do you must go to the source code of the service.
This feature should be documented for better knowledge.

### PR DESCRIPTION
**Pull request by sevdog**
Fixes #13049

### CODE REVIEW

File: src/ngAnimate/animateCss.js [hunk 1]
```diff
@@ -186,6 +186,8 @@ var ANIMATE_TIMER_KEY = '$$animateCss';
  *
  * * `event` - The DOM event (e.g. enter, leave, move). When used, a generated CSS class of `ng-EVENT` and `ng-EVENT-active` will be applied
  * to the element during the animation. Multiple events can be provided when spaces are used as a separator. (Note that this will not perform any DOM operation.)
+ * * `structural` - Indicates that the `ng-` prefix will be added to the event class. Setting `false` or omitting wil turn `ng-EVENT` and
```
Comment (gkalpak) [inline src/ngAnimate/animateCss.js]: _Setting \`false\`_  -->  _Setting to \`false\`_
_wil_  -->  _will_

**Comment by sevdog** [comment]
@googlebot I signed it!

**Comment by matsko** [comment]
Good work. Would you mind squashing everything into one commit and then rebasing on master?

**Comment by sevdog** [comment]
I will try, as soon as i understand how to do it ;)

**Comment by sevdog** [comment]
After some headache I think I'va made it!

**Comment by matsko** [comment]
Landed as https://github.com/angular/angular.js/commit/4038aabffa0a85639032e7f9330a3f8dee6d514f for master and https://github.com/angular/angular.js/commit/7bf5429e3b9f096e5d63e43a3fd1b33c0623bc8c for 1.4.x.

**Pull request closed without merging**